js/index.js

```diff
@@ -52,3 +52,21 @@ messageForm.addEventListener('submit', (event) => {
     messageList.appendChild(newMessage);
     messageForm.reset();
     });
+
+fetch('https://api.github.com/users/{kirt_desai}/repos')
```

Comment:
kirt_desai is undefined, and interpolation does not work with that type of quote, so this code will not work.